ui/views_test.py

```diff
@@ -72,6 +72,21 @@ def user_admin_list_data():
     return SimpleNamespace(video=video, moira_list=moira_list, collection=collection)
 
 
+@pytest.fixture
+def post_data(logged_in_apiclient):
+    """Fixture for testing collection creation using valid post data"""
+    client, user = logged_in_apiclient
+    url = reverse('models-api:collection-list')
+
+    input_data = {
+        'owner': user.id,
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
+    }
+    return SimpleNamespace(client=client, user=user, url=url, data=input_data)
```

Comment:
The client isn't really _data_. `post_data.client` is awkward to read and reason about, and I think it's important to not use fixtures to store all kinds of unrelated data. I would move that out of the return value namespace and have each test case just specify `logged_in_apiclient` in the args list. This is made a little bit confusing due to the multiple return values approach that `logged_in_apiclient` uses, but that can be fixed in a later PR. I would also move `url` out since it isn't data, and instead make that a class level variable (you can group these tests into a class since they're all testing the same view class). What do you think about something like this?:

```python
class CollectionListTests:
    url = reverse('models-api:collection-list')
    
    @pytest.fixture
    def input_data(self, logged_in_apiclient):
        _, user = logged_in_apiclient
        return {
            'owner': user.id,
            'title': 'foo title',
            'view_lists': [],
            'admin_lists': []
        }
    
    def test_collection_viewset_create_as_normal_user(logged_in_apiclient, input_data):
        client, user = logged_in_apiclient
        result = client.post(self.url, input_data, format='json')
        # ...
```

Let me know if you have any disagreements about the pattern in general